format/private/format.sh

```diff
@@ -122,7 +119,22 @@ function ls-files {
           fi
           find_args+=("-name" "${patterns[$i]}")
         done
-        find "$@" "${find_args[@]}"
+        files=$(find "$@" "${find_args[@]}")
+    fi
+
+    if [[ $disable_git_attribute_checks == true ]]; then
+      echo $files
+      return
+    fi
+
+    if [[ $files != "" ]]; then
```

Comment:
This part of change seems to cause quite some performance degradation for a big repo. I'm doing more tests to understand why.

Reply:
I think it is due to `echo "$files" | tr \\n \\0`